portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,232 @@
+          of a particular thyroid-cell slide, which was then fed into our classifier after it had been trained. We used Duke's massive GPU Computer Cluster to train our models in the interest of time. 
+          Throughout this project, we made multiple presentations, culminating in a final poster session.</p>
+        
+      </div>
+
+      <!-- This section includes my favorite places -->
+      <div id="places" class="section-content">
+        <h2>Places</h2>
+        <h3>The Gym</h3>
+        <img src="/images/workout.jpg" alt="The Gym" class="image-container" >
+        <p>Weightlifting is one of my hobbies; therefore, the gym is one of my favorite places to go to.
+          I enjoy working out and pushing myself to improve each time I go to the gym. 
+          Although sometimes I may lack the motivation, I always come out of the gym feeling good 
+          knowing that I had accomplished something that will inevitably improve my health and well-being. 
+          Going with my friends motivates me a lot as we can push each other and hold each other accountable.
+          When I put 250 pounds on my back and perform squats, I feel powerful and motivated to succeed in other endeavors.</p>
+        <h3>Duke University</h3>
+        <img src="/images/Duke.JPG" alt="Duke's Campus" class="image-container">
+        <p>Duke University's campus is one of my favorite places on Earth. Duke feels like a completely different place, as if
+          it is in its own bubble. The architecture is majestic and truly breathtaking. The facilities are packed with resources:
+          I have access to over 40 local restaurants on campus alone, two enormous gym facilities, countless places to study,
+          multiple libraries, about a dozen graduate and professional schools, and world-class professors. Although the overwhelming
+          amount of resources available to is a huge draw, I feel like the people is what makes Duke such a great place. I've made so 
+          many amazing friends throughout my two years there and I have made so many valuable connections along the way. 
+          It's like home away from home.</p>
+      </div>
+
+      <!-- This section includes paragraphs about things I like photographing, along with associated photographs-->
+      <div id="photographing" class="section-content">
+        <h2>Photographing</h2>
+        <p>Forza Horizon 4 is an open-world racing game set in Great Britain. It is perhaps one of the most beautiful and fun racing games currently available.
+          It is so technically impressive in fact, that a Photo Mode is included which allows the player to take amazing photographs of cars in the breathtaking vistas of the UK.
+          Here I have included some photos taken in-game:</p>
+        <div id =fh4-photographs>
```

Comment:
This is another good opportunity to make some sort of component to build this for you. You could write a generic JS function that would take a config with your photo params and build this for you. Again, not required right now, but consider adding a bug and TODO to improve this.